test/e2e/parallel/nstemplatetier_test.go

```diff
@@ -455,10 +455,20 @@ func TestTierTemplateRevision(t *testing.T) {
 			// then
 			// an additional TTR will be created
 			// TODO check for exact match or remove the +1 once we implement the cleanup controller
-			ttrsWithNewParams, err := hostAwait.WaitForTTRs(t, customTier.Name, wait.GreaterOrEqual(len(updatedTTRs)+1))
+			ttrsWithNewParams, err := hostAwait.WaitForTTRs(t, customTier.Name, wait.GreaterOrEqual(2*len(ttrs)+1))
```

Comment:
why `*2` ? Unless I'm missing something it should only trigger creation of 1 TTR ( in most cases ) .

Also I would say that , we should move this check after the wait for NSTemplateTier at line 461, so that the retrieved TTRs are with the new parameter.

Reply:
this changes is triggered based on the change in the list of parameters in NSTemplateTier CR, which mean that it should create new TTR CR for each TierTemplate referenced from the NSTemplateTier, right?

Reply:
you're right! 👍 

I still think this is valid: 
> Also I would say that we should move this check after the wait for NSTemplateTier at line 461, so that the retrieved TTRs are with the new parameter.

Reply:
:man_facepalming: fixed in [dc6c89f](https://github.com/codeready-toolchain/toolchain-e2e/pull/1116/commits/dc6c89f3df75e6342b582d212a016887945f957c)